src/spetlr/tables/TableHandle.py

```diff
@@ -49,5 +49,11 @@ def get_table_id(self):
     def get_schema(self) -> T.StructType:
         raise NotImplementedError()
 
-    def set_schema(self, schema: T.StructType) -> T.StructType:
+    def set_schema(self, schema: T.StructType) -> "TableHandle":
```

Comment:
Just curious. 

1. Do we have a great need for these getters and setters? Personally, i couldn't see where i would use them. 

3. What is the reasoning behind the design choice of not using the `property` function/decorator with such behaviour?

Reply:
1. Yes. The get/set schema is used in the schema validation extractor. I want get/set options in cases where the handle is created from table config, but some extra options is set.
2. Could you give an example of what you mean by the property decorator.